requirements/requirements_base.txt

```diff
@@ -1 +1,2 @@
-async-worker==0.11.1
\ No newline at end of file
+async-worker==0.11.1
+aioamqp==0.14.0
```

Comment:
This shouldn't be necessary anymore

Reply:
It is... Just new versions of async-worker are covered